MachineLearning_Engine/Compute/MachineLearningToolkitPythonEnvironment.cs

```diff
@@ -20,28 +20,31 @@
  * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
  */
 
-using BH.oM.Base.Attributes;
+
+using BH.oM.Python;
 using System.ComponentModel;
 
-namespace BH.Engine.MachineLearning.Vision
+using BH.oM.Base.Attributes;
+
+namespace BH.Engine.MachineLearning
 {
     public static partial class Compute
     {
-        /*************************************/
-        /**** Public Methods              ****/
-        /*************************************/
-
-        [Description("Returns the type of object in the image, e.g. an apple")]
-        [Input("Image path", "The absolute path to the image to analyse")]
-        [Input("Gpu", "If true, computation is accelarated using a capable gpu")]
-        [Output("Object", "The probabilities (score) for each class. Use the Query.ImageClasses() method to match ")]
-        public static string RecogniseObject(string imagePath, bool gpu = false)
+        [Description("MachineLearning_Toolkit\nMethod used to create the Python environment used to run all Python scripts within this toolkit.")]
+        [Input("run", "Starts the installation of the toolkit if true. Stays idle otherwise.")]
+        [Output("pythonEnvironment", "The MachineLearning_Toolkit Python environment.")]
+        public static PythonEnvironment MachineLearningToolkitPythonEnvironment(bool run = false)
         {
-            return BH.Engine.MachineLearning.Base.Compute.Invoke(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType.Namespace, "RecogniseObject.infer", imagePath, gpu).As<string>();
+            if (run)
+            {
```

Comment:
Please remove indented block. Perhaps consider adding a warning to alert user to toggle `run` to `true`.
```suggestion
            if (!run)
            {
               BH.Engine.Base.Compute.RecordWarning($"Please toggle `{nameof(run)}` to true.");
               return null;
            }
```

Reply:
Just realised that changes in the Python_Tk should mean that we don't have to `if(run)` check any of the dependant methods.